### ISSUE TITLE
getSubnetsAndAvailZones breaks if EC2.DescribeSubnet returns nil

### ISSUE THREAD

**Issue by tlake** [quintilesims/layer0#100]
Within `api/backend/ecs/load_balancer_manager.go: getSubnetsAndAvailZones()`, if `e.EC2.DescribeSubnet()` returns a `nil` value, Layer0 breaks with the following error when it tries to dereference that `nil` value with `if *description.AvailabilityZone == *zone {`:
```
Received: 500 Internal Server Error [restful] recover from panic situation: - runtime error: invalid memory address or nil pointer dereference
```

**Comment by tlake** [comment]
_@zpatrick commented in slack:_

> That seems like a check we are missing in the  `layer0/common/aws` package
> in the ` e.EC2.DescribeSubnet(subnetID)` function

**Comment by tlake** [comment]
Update: it looks like this error arose from a badly-set `LAYER0_AWS_PRIVATE_SUBNETS` environment variable:
```
subnet-2d4ee14a, subnet-11f83958
```
instead of
```
subnet-2d4ee14a,subnet-11f83958
```
The space after the comma causes the `common/aws/ec2/ec2.go: DescribeSubnet(" subnet-11f83958")` call to return `nil` instead of a subnet description.

**Comment by tlake** [comment]
Fixed with #101 #102
